custom.props

```diff
@@ -4,7 +4,7 @@
     <VersionMajor>1</VersionMajor>
     <VersionMinor>2</VersionMinor>
     <!-- The nuget package version should be incremented when we produce QFEs -->
-    <NuGetPackVersion>1.2.0-beta1</NuGetPackVersion>
+    <NuGetPackVersion>1.2.1</NuGetPackVersion>
```

Comment:
Wy do we need to increment the AC package version? I thought this was a new package we were spinning up?